website/docs/cloud-docs/run/states.mdx

```diff
@@ -79,7 +79,7 @@ Note, if you want to directly integrate third-party tools and services between y
 
 ## The Post-Plan Stage
 
-The post-plan phase executes any configured run tasks after the plan is complete, so it only occurs if there are [run tasks](/terraform/cloud-docs/workspaces/settings/run-tasks) enabled
+The post-plan phase executes any configured run tasks after the plan is successfully complete, so it only occurs if there are [run tasks](/terraform/cloud-docs/workspaces/settings/run-tasks) enabled
```

Comment:
```suggestion
The post-plan phase executes any configured run tasks after the plan is successful, so it only occurs if there are [run tasks](/terraform/cloud-docs/workspaces/settings/run-tasks) enabled
```

Is there a difference between _successful_ and _successfully complete_? If so and we desire _successfully complete_ perhaps it can be _after the plan has successfully completed_.

Reply:
Hrmmm interesting question ...

Originally, I thought the main difference (I see) is that we use the term "complete" frequently in documentation.  BUT we do say "A plan was succesful". So yes, I think that is a valid way of describing it!

Then, is "plan is succesful" less or more readable than "plan is successfully complete". I'd say it's the same and more succinct. 

And finally, should it be in present or past tense. The sentence is talking about  "post-plan executes after ..." so it should be past tense.

i.e.`The post-plan phase executes any configured run tasks after a successful plan`,

Is that better?

Reply:
Oh, and I'll apply the same wording to the post-apply section too.

Reply:
In documentation we always prefer present tense and active voice for clarity! 

I'm not sure if this applies here, but the main difference between "complete plan" and "successful plan", is the the case where a plan errors and cannot finish successfully. You could argue that a failed plan is still "complete", but not "successful". 

How would you feel about a suggestion like below? Echoing the language from the post-apply phase:
```suggestion
The post-plan phase only occurs if you configure [run tasks](/terraform/cloud-docs/workspaces/settings/run-tasks) on a workspace to begin after Terraform successfully completes a plan operation.
```

Reply:
That sounds good to me Rose.  Thankyou!

@dylanegan you ok if I use that wording too?

Reply:
👍 thank you so much @rkoron007. 🙇‍♂️

Reply:
Done.